rest-api/src/main/resources/definitions/oauth_provider.yml

```diff
@@ -22,6 +22,10 @@ properties:
         description: |
             The callback URL that the client provides to retrieve the authorization token. This must match that callback URL exactly.
         x-nullable: false
+    introspectEndpoint:
+        type: string
+        description: OAuth introspect URL, as defined by RFC7662. Optional but recommended.
+        x-nullable: true
```

Comment:
Maybe document that the only information we pull from this endpoint are the scopes (and that in Fitbit's non-standard format)?

Reply:
Updated comments.